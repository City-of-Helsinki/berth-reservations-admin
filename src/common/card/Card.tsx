import React from 'react';
import classNames from 'classnames';

import styles from './card.module.scss';

export interface CardProps {
  children: React.ReactNode;
  className?: string;
}

const Card: React.SFC<CardProps> = ({ children, className }) => (
  <div className={classNames(styles.card, className)}>{children} </div>
);

export default Card;
