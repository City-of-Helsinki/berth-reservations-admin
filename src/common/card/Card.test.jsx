import { shallow } from 'enzyme';
import React from 'react';

import Card from './Card';

describe('common/Card', () => {
  const getWrapper = (props) => {
    return shallow(<Card {...props}>placeholder</Card>);
  };
  it('renders a Card component', () => {
    const card = getWrapper();
    expect(card.html()).toMatchSnapshot();
  });
  it('renders a Card component without title', () => {
    const card = getWrapper();
    expect(card.html()).toMatchSnapshot();
  });
});
