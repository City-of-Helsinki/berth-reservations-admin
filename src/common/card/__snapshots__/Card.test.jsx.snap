// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/Card render Card component 1`] = `"<div style=\\"width:100%\\"><div class=\\"container\\"><h1>dummy</h1></div></div>"`;
